src/execution_service/infrastructure/docker_client/DockerClient.ts

```diff
@@ -0,0 +1,90 @@
+import * as Docker from 'dockerode'
+import { Contract } from '../../../core'
+import { DockerExecutionEngineContext } from '../execution_engines'
+
+interface Config {
+  dockerOptions?: Docker.DockerOptions | { socketPath: '/var/run/docker.sock' },
+  executionContext: DockerExecutionEngineContext,
+  pipeStdout?: boolean | true
```

Comment:
Were you trying to achieve something with the `| true` here? `true` is on the `boolean` enum, so from TypeScripts perspective, this doesn't do anything

Reply:
ah good catch. Was intending to be applying a default value. I'm going to make it explicit and required